<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeadWidget</class>
 <widget class="QWidget" name="HeadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>409</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hBox1">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>游客，你好！</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutBtn">
      <property name="text">
       <string>注销</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="checkBtn">
      <property name="text">
       <string>查看信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mineBtn">
      <property name="text">
       <string>查看我的商品</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addItemBtn">
      <property name="text">
       <string>添加商品</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hBox2">
    <item>
     <widget class="QLabel" name="tourLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>游客，你好！</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginBtn">
      <property name="text">
       <string>登陆</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
